src/components/tiptapEditor/tiptap.jsx

```diff
@@ -272,7 +314,44 @@ export default function Tiptap({ provider, ydoc, isInlineEditor, disabled, initi
 
       {showSlashMenu && <ShowSlashMenuComponent editor={editor} pathData={pathData} pathName={pathName} showSlashMenu={showSlashMenu} setShowSlashMenu={setShowSlashMenu} searchQuery={searchQuery} setSearchQuery={setSearchQuery} activeSlashMenuIndex={activeSlashMenuIndex} setActiveSlashMenuIndex={setActiveSlashMenuIndex}  slashMenuPosition={slashMenuPosition} showImage={showImage} setShowImage={setShowImage} showVideo={showVideo} setShowVideo={setShowVideo} showFiles={showFiles} setShowFiles={setShowFiles} />}
 
-      <EditorContent editor={editor} />
+      <EditorContent editor={editor} onClick={(event) => {
+          if (!event.target.classList.contains("callout-icon")) {
+            setModalVisible(false); 
+          }
+        }} />
+        
+        {modalVisible && (
+          <div
+            className="icon-modal"
```

Comment:
use bootstrap class position-fixed border rounded p-2 
box-shadow and z-index and background ka varibles bna hai white color ka usko use kariye usko  class ka nam likhke iske scss file me likh dijiye baki yaha serf top  left bs style me likhiye baki class use kariye
```suggestion
            className="icon-modal  position-fixed border rounded p-2 "
```

Reply:
Resolved